README.adoc

```diff
@@ -385,6 +385,24 @@ class Transaction < ActiveRecord::Base
 end
 ----
 
+=== Enums [[enums]]
```

Comment:
There's this example:
```ruby
# Rails 4+ enums after attr macros, prefer the hash syntax
```

Do you think it becomes redundant with this new guideline?

Reply:
Yes, I saw the guideline exists (hence placed this rule just above). I feel the comment reiterates the rule & don't see much of an issue. I can remove `, prefer the hash syntax` if required.

Reply:
Removed.